nipy/fixes/scipy/ndimage/__init__.py

```diff
@@ -0,0 +1,53 @@
+""" Patches for scipy.ndimage
+
+Patched affine_transform to work round np.intp bug.
+
+Some versions of scipy.ndimage interpolation routines can't handle the np.intp
+type. I (MB) have only seen this on a 32-bit machine runing scipy 0.9.0
+"""
+
+import numpy as np
+
+import scipy.ndimage as spnd
+
+# int dtype corresponding to intp
+_INT_DTYPE = np.dtype(np.int32 if np.dtype(np.intp).itemsize == 4 else
+                      np.int64)
+
+
+# Do not run doctests on this module via nose; scipy doctests unreliable
+__test__ = False
+
+
+def _proc_array(array):
+    if array.dtype == np.dtype(np.intp):
+        return array.view(_INT_DTYPE)
+    return array
+
+
+def _proc_output(output):
```

Comment:
In case it matters, I find this function a bit suprising.
In particular
```
if np.dtype(output) == np.dtype(np.intp):
        return _INT_DTYPE
```
Since it returns a dtype ?
Maybe a short explanation is needed.